### ISSUE TITLE
Step 2 Warning before installing Docker (VM problem) after performing Step 1

### ISSUE THREAD

**Issue by JackBerezhnov** [open-learning-exchange/open-learning-exchange.github.io#2126]
### Problem
Your VM VirtualBox must be completely disabled before installing Docker.
People don’t give proper attention or skip this step because it’s too small in the Docker installation tutorial.

### Screenshots
![issue](https://user-images.githubusercontent.com/33531079/51011266-bdd2a400-150c-11e9-9a22-032bc45927d1.png)

### Proposed solution
Add an entry with the note "that your VM VirtualBox should be completely turned off before starting the installation and do not proceed to the instructions below until this is done."

**Comment by lmmrssa** [comment]
@JackBerezhnov sorry I don't understand why VM needs to be completely disabled.
Yes if you are trying to run Windows docker by Hyper-V then VM can not run along with it. But we prefer to use Dockertool for that which run with VM technology so that VM and Docker can run alongside.

**Comment by JackBerezhnov** [comment]
I apologize if my thought was not pure. I mean, when people launch vagrant in step 1 and then go to the second step, they forget to turn off the VM before installing the Docket tool and in their instructions they say it .That's what I'm trying to say.

![dockerissue](https://user-images.githubusercontent.com/33531079/51012113-5e769300-1510-11e9-88a3-13744c2e01c9.png)

**Comment by YuserahN** [comment]
@JackBerezhnov @lmmrssa 
When I tried to install Docker, I ran into some issues and that may be what Jack Berezhnov is referring to. When installing Docker, VM Virtual Box is part of the package, as you know. Therefore, in the Setup Wizard for Docker Toolbox, it is important to uncheck the box next to "install VM" if you already have VM installed on your computer. While that should work in theory, it did not for me. Neither did disabling VM. I still ran into errors while installing Docker. It was resolved when I uninstalled VM and then installed Docker with VM as part of the package.
